python/README.md

```diff
@@ -30,13 +30,18 @@ AZURE_OPENAI_ENDPOINT=""
 
 ```python
 import semantic_kernel as sk
+import semantic_kernel.ai.open_ai as sk_oai
 
 kernel = sk.create_kernel()
 
+# This requires a `.env` file in your current
+# directory (see above)
 api_key, org_id = sk.openai_settings_from_dot_env()
 
-kernel.config.add_openai_completion_backend(
-    "davinci-002", "text-davinci-002", api_key, org_id
+kernel.config.add_text_backend(
+    "davinci-002", sk_oai.OpenAITextCompletion(
+        "text-davinci-002", api_key, org_id
+    )
 )
 
 sk_prompt = """
```

Comment:
module 'semantic_kernel' has no attribute 'extensions'.

According to the python notebooks, it should look like 

```
tldr_function = kernel.create_semantic_function(
    sk_prompt,
    max_tokens=200,
    temperature=0,
    top_p=0.5,
)
```

Reply:
Thank you! Another great catch, I fixed README first but realized (fixing notebooks) how clunky that syntax was, and then never re-tested README.